### ISSUE TITLE
Docs suggestion: How to use rule outputs for directories cached on Zenodo

### ISSUE THREAD

**Issue by LouisDesdoigts** [showyourwork/showyourwork#246]
In the [Zenodo section](https://show-your.work/en/latest/zenodo/) of the docs it shows how to cache a directory to Zenodo, but doesn't show how to update the `Snakefile` and `showyourwork.yml` to access these directories appropriately. I ended up looking through the projects that use showyourwork to find out how to do this, ie

In Snakefile
```
rule:
    input:
        rules.some_other_rule.output
```

and in the `showyourwork.yml`
```
dependencies:
  src/scripts/some_script.py:
    - src/data/some_directory/
```

I think this could be quite valuable for other users like me who are new to snakemake etc. 

I would be happy to make this update to the docs myself in a branch and submit it for review, so let me know you think that would be helpful!

Cheers

**Comment by dfm** [comment]
Is that not what is described here: https://show-your.work/en/latest/zenodo/#the-streamlined-version

What is missing from that discussion and where do you think it should go? I'd be very happy to review a PR that makes this clearer!

**Comment by dfm** [comment]
Closed by #248, I think!
